src/adm/settings.rs

```diff
@@ -21,12 +21,15 @@ pub struct AdmAdvertiserFilterSettings {
     /// Set of valid hosts for the `advertiser_url`
     pub(crate) advertiser_hosts: Vec<String>,
```

Comment:
Do we need a `#[serde(default)]` tag for this attribute as well?

Reply:
No. While the other values are going to be optional (and thus potentially fall back to a `Default`) we want these to be expressly defined. I can improve the comment descriptor to indicate that.